test/e2e/clusterctl_upgrade_test.go

```diff
@@ -25,40 +25,6 @@ import (
 	"sigs.k8s.io/cluster-api/test/framework"
 )
 
-var _ = Describe("When testing clusterctl upgrades (v0.3=>current)", func() {
```

Comment:
My bad. Let's please also adjust our docs. Should be at least: release-tasks, book where we have the test table